### ISSUE TITLE
FR: set colors via theme variables 

### ISSUE THREAD

**Issue by darthsebulba04** [dbuezas/lovelace-plotly-graph-card#54]
I have different themes on every device so it would be great to be able to set line colors (and others?) to theme variables to make everything look natural. 

More details on the forum post below. Using the format `var(—background-color)` is the proposed way to express it. 

https://community.home-assistant.io/t/plotly-interactive-graph-card/347746/110

Thanks!

**Comment by dbuezas** [comment]
`css_vars` are now passed as argument to universal functions (`$fn`), so your request is finally possible to implement in yaml :) 
Closed by the release of v3.0.0

**Comment by simenopsahl** [comment]
It'd be nice to have access to more colors through `css_vars`
Specifically:
```ts
    /* color */
    --disabled-color: #bdbdbd;
    --red-color: #f44336;
    --pink-color: #e91e63;
    --purple-color: #926bc7;
    --deep-purple-color: #6e41ab;
    --indigo-color: #3f51b5;
    --blue-color: #2196f3;
    --light-blue-color: #03a9f4;
    --cyan-color: #00bcd4;
    --teal-color: #009688;
    --green-color: #4caf50;
    --light-green-color: #8bc34a;
    --lime-color: #cddc39;
    --yellow-color: #ffeb3b;
    --amber-color: #ffc107;
    --orange-color: #ff9800;
    --deep-orange-color: #ff6f22;
    --brown-color: #795548;
    --light-grey-color: #bdbdbd;
    --grey-color: #9e9e9e;
    --dark-grey-color: #606060;
    --blue-grey-color: #607d8b;
    --black-color: #000000;
    --white-color: #ffffff;
```
In `ha_style.ts`
So when those colors are modified by themes, the charts update.

<img width="653" height="728" alt="Image" src="https://github.com/user-attachments/assets/872bba1b-b5d5-4164-9676-a6ed7bed12ab" />

I use the Material You theme, and get the overall theme color from the Android phone accent color (or the Windows accent color depending on device used).

Just the `primary-color` and the `primary`/`secondary`-`text-color` are limiting when using multiple traces

🙏🙏
